src/main/java/dev/dokan/dokan_java/structure/filesecurity/SecurityIdentifier.java

```diff
@@ -38,7 +36,7 @@ public SecurityIdentifier(SidIdentifierAuthority sidAuth, List<Integer> subAutho
 			if (subAuthorities.size() <= 15) {
 				this.subAuthorities.addAll(subAuthorities);
 			} else {
-				LOG.warn("Number of subauthorities exceeds the limit of 15. Only taking the first 15 ones.");
+				//LOG.warn("Number of subauthorities exceeds the limit of 15. Only taking the first 15 ones.");
```

Comment:
In my opinion there _should_ be a warning. Maybe we need to throw an exception here.

Reply:
Ok. This is the only reason to have dependency to Sl4J, maybe we can do something else without requireing SLF4J?